cid/builtin/core/data/queries/shared/account_map_dummy.sql

```diff
@@ -1,10 +1,14 @@
-CREATE OR REPLACE VIEW  ${athena_view_name} AS
+CREATE OR REPLACE VIEW ${athena_view_name} AS
 SELECT DISTINCT
     line_item_usage_account_id                                     account_id,
-    MAX_BY(bill_payer_account_id,      line_item_usage_start_date) parent_account_id,
     MAX_BY(line_item_usage_account_id, line_item_usage_start_date) account_name,
-    MAX_BY(line_item_usage_account_id, line_item_usage_start_date) account_email_id
+    MAX_BY(bill_payer_account_id,      line_item_usage_start_date) parent_account_id,
+    CAST('' AS varchar)                                            account_status,
+    CAST('' AS varchar)                                            account_email,
+    CAST('' AS varchar)                                            account_email_id
```

Comment:
why we need both email_id and email?

Reply:
I am just trying to be very careful here. Email_id is not really used but I don't know if it was used somewhere in the past and if therefore we need some backwards compatibility.
But if we agree that this is not a valid concern, I would be happy to get rid of it.

Reply:
email_id is removed